python/cudf/cudf/tests/test_numerical.py

```diff
@@ -64,6 +67,30 @@ def test_can_cast_safely_mixed_kind():
     assert not data.can_cast_safely(to_dtype)
 
 
+@pytest.mark.xfail
```

Comment:
Let's use the `reason=` argument to `xfail` here to describe why we are expecting to fail this test.